testCourse/courseInstances/Sp15/infoCourseInstance.json

```diff
@@ -1,7 +1,8 @@
 {
   "uuid": "31a69dd9-81a1-4b17-b578-2098d20fb57a",
   "longName": "Spring 2015",
-  "allowAccess": [
-    { "institution": "Any", "startDate": "1900-01-19T00:00:01", "endDate": "2400-05-13T23:59:59" }
-  ]
+  "publishing": {
```

Comment:
The example course instance and public were intentionally not converted. The former needs the feature flag in prod, and I use the latter for access rules in a API test.

Reply:
Not that it should really matter, but the removal of `"institution": "Any"` without a replacement to allow self-enrollments from non-institution users is a change in behavior.

It shouldn't matter for the tests, so I don't think we need to do anything about this.